samcli/commands/local/lib/debug_context.py

```diff
@@ -0,0 +1,17 @@
+"""
+Information and debug options for a specific runtime.
+"""
+
+
+class DebugContext(object):
+
+    def __init__(self,
+                 debug_port=None,
+                 runtime=None,
+                 debugger_path=None,
+                 debug_args=None):
+
+        self.debug_port = debug_port
+        self.runtime = runtime
```

Comment:
so `runtime` is unused, right? can we remove it?

Reply:
It's currently unused, yeah, I can pop it out of there.